README.md

```diff
@@ -183,7 +183,7 @@ procedure Main is
    Buffer  : RFLX.RFLX_Builtin_Types.Bytes_Ptr := new RFLX.RFLX_Builtin_Types.Bytes'(1, 0, 4, 0, 0, 0, 0);
    Context : RFLX.TLV.Message.Context;
 begin
-   RFLX.TLV.Message.Initialize (Context, Buffer);
+   RFLX.TLV.Message.Initialize (Context, Buffer, Written_Last => RFLX.RFLX_Types.To_Last_Bit_Index (Buffer'Last));
```

Comment:
Shouldn't the signature of the `Initialize` procedure in the list above also be adapted?

Reply:
Yes, I have fixed that.